src/modules/fancyzones/lib/ZoneWindow.cpp

```diff
@@ -106,7 +157,7 @@ struct ZoneWindow : public winrt::implements<ZoneWindow, IZoneWindow>
     void CalculateZoneSet() noexcept;
     void UpdateActiveZoneSet(_In_opt_ IZoneSet* zoneSet) noexcept;
     LRESULT WndProc(UINT message, WPARAM wparam, LPARAM lparam) noexcept;
-    void OnPaint(wil::unique_hdc& hdc) noexcept;
+    void OnPaint(HDC hdc) noexcept;

```

Comment:
What is the reason to change `unique_hdc` to `HDC`?

Reply:
So that I can create a copy of it in the lambda function at line 643, since you can't copy a `unique_hdc`. And also because it seems simpler overall.

Reply:
According to the documentation, `unique_hdc` frees the DC with `DeleteDC`. I suppose `DeleteDC` should be called manually now.

Reply:
No, because of `hdc.release();`, that function returns the stored HDC and resets its internal HDC, so `DeleteDC` won't be called.